app/models/concerns/visible.rb

```diff
@@ -1,18 +1,20 @@
-module Visible
-    extend ActiveSupport::Concern
-
-    VALID_STATUSES = ['public', 'private', 'archived']
-
-    included do
-        validates :status, inclusion: { in: VALID_STATUSES }
-    end
-    class_methods do
-        def public_count
-            where(status: 'public').count
-        end
-    end
-    
-    def archived?
-        status == 'archived'
-    end
-end
\ No newline at end of file
+# frozen_string_literal: true
```

Comment:
What is the function of this line?

Reply:
This helps to transition from mutable string to frozen string. It is important because in line 6 of the same code the variable `VALID_STATUSES` has been freeze